index.js

```diff
@@ -123,6 +123,25 @@ class OError extends Error {
     return info
   }
 
+  /**
+   * The merged info from any `tag`s and nested causes on the given error.
+   *
+   * See also `OError.getFullInfo`.
+   *
+   * @param {Error | null | undefined} error any error (may or may not be an `OError`)
+   * @return {Object}
+   */
+  static getFullInfoIncludeCause(error) {
+    const info = OError.getFullInfo(error)
+    if (!error) return info
+
+    const oError = /** @type{OError} */ (error)
+    if (oError.cause) {
+      Object.assign(info, OError.getFullInfoIncludeCause(oError.cause))
```

Comment:
A little detail: written this way, if a property is present both in the main error and in a cause, the property value on the cause wins. We probably want to have it the other way around.

Reply:
I copied the behavior from `getFullInfo()`, which may get context details overwritten from tags.